kubernetes/helm/prometheus/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: prometheus
+description: Prometheus operator
+version: 0.1.0
+appVersion: 0.1.0q
```

Comment:
This appVersion seems wrong.

Reply:
Yeah, that q was when doing something with vim. I'll fix that.